oca_dependencies.txt

```diff
@@ -0,0 +1,16 @@
+# List the OCA project dependencies, one per line
+# Add a repository url and branch if you need a forked version
+#
+# Examples
+# ========
+#
+# To depend on the standard version of sale-workflow, use:
+# sale-workflow
+#
+# To explicitely give the URL of a fork, and still use the version specified in
+# .travis.yml, use:
+# sale-workflow https://github.com/OCA/sale-workflow
+#
+# To provide both the URL and a branch, use:
+# sale-workflow https://github.com/OCA/sale-workflow branchname
+account-financial-tools https://github.com/rven/account-financial-tools 13.0-mig-account_menu
```

Comment:
```suggestion
account-financial-tools
```

If this PR depends on another, we should wait for the other to be merged first.
In this case I think it's already merged.

You can remove the comments on this file aswell